app/api-v1/routes/self.js

```diff
@@ -7,11 +7,10 @@ const { SELF_ADDRESS } = env
 export default function (apiService) {
   const doc = {
     GET: async function (req, res) {
-      const [self] = await apiService.getMembersByAddress(SELF_ADDRESS)
-      const response = self || { address: SELF_ADDRESS, alias: null }
-      const errors = validateSelfResponse(400, response)
+      const self = await apiService.getMemberByAddress(SELF_ADDRESS)
+      const errors = validateSelfResponse(400, self)
```

Comment:
why do we pass 400 and return 400? just wondering and lazy to investigate - if you do not know, please do not answer

Reply:
from using https://www.npmjs.com/package/openapi-response-validator , it's validate function requires a status code